src/about-menu/about-menu.html

```diff
@@ -73,10 +73,10 @@
     line-height: 20px;
   }
 </style>
-
 </head>
 <body> 
-  <div>  
+  <div>
+    <!-- When updating these strings, make sure to update the corresponding strings in the open-about-menu.ts file -->
```

Comment:
Can we say more explicitly that "these strings are placeholders replaced by `open-about-menu.ts`" ?

Reply:
Sure, I made updates in https://github.com/Automattic/studio/pull/290/commits/7887d3f46ae701aff34962c52638f3088f418c19